cli/tests/integrations/test_analytics.py

```diff
@@ -115,64 +173,6 @@ def test_config_reporting_false():
         assert requests.post.call_count == 0
 
 
-@_mock
-def test_production_setting_true():
```

Comment:
This error raises in every test after starts at `test_analytics` module: 
https://teamcity.mesosphere.io/viewLog.html?buildId=21743&buildTypeId=ClosedSource_DcosCli_WindowsTests_WindowsTest&tab=buildLog#_focus=23637